demo/package.json

```diff
@@ -10,7 +10,8 @@
   },
   "dependencies": {
     "@ant-design/icons": "^5.4.0",
-    "@figmentio/elements": "~1.3.0",
+    "@figmentio/elements": "~1.4.2",
+    "@solana/web3.js": "~1.95.4",
```

Comment:
Thoughts on using v2 of this package?

Reply:
Sure - I'll make sure we can use it the same way in v2.